File: _old_CmdMents/TimesAndSeasons/RestOnYomTeruah.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CmdMents.Torah;

namespace CmdMents.TimesAndSeasons
{
    class RestOnYomTeruah : SoundTheShofarOnYomTeruah
    {
        public RestOnYomTeruah()
        {
            base.AlternateText = null;
            base.Book = CommandmentBook.Leviticus;
            base.CanBeCarriedOutOnlyInIsrael = false;
            base.CanBeCarriedOutToday = true;
            base.Chapter = 23;
            base.CommandmentType = CmdMents.CommandmentType.Positive;
            base.FollowedByChristians = CommandmentObedience.None;
            base.FollowedByMessianics = CommandmentObedience.Obeyed;
            base.FollowedByObservantJews = CommandmentObedience.Obeyed;
            base.Number = 102;
            base.ShortSummary = "Rest on Yom Teruah.";
            base.Text = "On the first day of the seventh month you are to have a day of rest, a sacred assembly commemorated with trumpet blasts.";
            base.Verse = 24;
        }
    }
}

File: EtzMitzvot/Commandments/TimesAndSeasons/restOnYomTeruah.ts
///<reference path='../commandmentBase.ts' />

class RestOnYomTeruah extends CommandmentBase {
    constructor() {
        super();

        this.shortSummary = "Rest on Yom Teruah";
        this.text = "On the first day of the seventh month you are to have a day of rest, a sacred assembly commemorated with trumpet blasts.";
        this.book = TorahBook.Leviticus;
        this.chapter = 23;
        this.verse = 24;

        this.canBeCarriedOutToday = true;
        this.requiresLivingInIsrael = false;
        this.requiresTemple = false;

        this.type = CommandmentType.Positive;
        this.christianObservance = CommandmentObedience.None;
        this.messianicObservance = CommandmentObedience.Binding;
        this.jewishObservance = CommandmentObedience.Binding;
        this.number = 102;

        this.children = [
            new DontWorkOnYomTeruah()
        ];
    }
}  